app/views/general_shopping_lists/general.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="container flex food-placeholder">
+<div class="food flex">
+  <h3 class="title">General Shopping List</h3>
+  <div class="sum flex">
+    <p>Amount of food to buy: <%= @general_shopping_lists.count %> </p>
+    <p>Total value of food needed: $<%= @general_shopping_lists.map { |food| food.price}.sum %></p>
+  </div>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Price</th>
+      </tr>
+    </thead>
+    <tbody>
```

Comment:
User should be allowed to generate a shopping list when he clicks on `Generate shopping list` but it takes the user to Recipes index page when clicking on the button. Kindly see the requirements for Recipe show page and fullfil them accordingly.